### ISSUE TITLE
Module to design siRNAs

### ISSUE THREAD

**Issue by tjakobi** [dieterich-lab/circtools#24]
A module to design siRNA sequences for use with circlular RNA was suggest on the SPP1738 conference and seems like a natural extension of circtools' functionality.

**Comment by tjakobi** [comment]
Note: take care of off-target effects, siRNA should be unique

**Comment by CDieterich** [comment]
https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkw1325

**Comment by CDieterich** [comment]
I would like to push this to Version 0.3.0

Do you agree ?

**Comment by tjakobi** [comment]
Yes, we agreed to push this module, just forgot to update the issue correspondingly.

**Comment by tjakobi** [comment]
Now under review in #81
